app/views/user_mailer/admin_hidden_work_notification.text.erb

```diff
@@ -1,12 +1,13 @@
 <% content_for :message do %>
-    Dear <%= @user.default_pseud.byline %>,
+<%= t(".hello", name: @user.login) %>
 
-    Your work "<%= @work.title.html_safe %>" (<%= work_url(@work) %>) has been reported to the Abuse Team as a violation of the Archive of Our Own's Terms of Service (<%= root_url + "tos" %>).
+<%= t(".text.hidden", title: @work.title, work_url: @work) %>
```

Comment:
Ah, found the culprit! This needs to be `work_url: work_url(@work)` to fix [the issue we found on staging](https://otwarchive.atlassian.net/browse/AO3-6206?focusedCommentId=357675).

Reply:
oh no... how does it look now

Reply:
https://github.com/otwcode/otwarchive/pull/4059/